app/controllers/miq_ae_class_controller.rb

```diff
@@ -2554,14 +2554,12 @@ def domain_overrides
   end
 
   def get_session_data
-    @layout     = "miq_ae_class"
     @title      = _("Datastore")
     @lastaction = session[:aeclass_lastaction]
     @edit       = session[:edit]
   end
 
   def set_session_data
-    session[:aeclass_lastaction] = @lastaction
```

Comment:
You will have to leave this line in or `@layout = "miq_ae_class"` as they both use `prefix` in mixin but expect different string `"miq_ae_class" != "aeclass"`. I will leave it up to you. But don't forget `super` :)